contracts/hardhat.config.ts

```diff
@@ -71,19 +100,28 @@ const config: HardhatUserConfig = {
   },
   namedAccounts: {
     sequencer: {
-      default: 0,
-      goerli: SEQUENCER_ADDRESS,
-      chiado: SEQUENCER_ADDRESS,
+      default: SEQUENCER_ADDRESS,
+      hardhat: 0,
+      localhost: 0,
+      specularLocalDev: 0,
     },
     validator: 1,
-    deployer: 2,
+    deployer: {
+      default: DEPLOYER_ADDRESS,
+      hardhat: 2,
+      localhost: 2,
+      specularLocalDev: 2,
+    },
   },
   networks: {
     mainnet: createConfig("mainnet"),
     sepolia: createConfig("sepolia"),
     goerli: createConfig("goerli"),
     gnosis: createConfig("gnosis"),
     chiado: createConfig("chiado"),
+    localhost: createConfig("localhost"),
+    specularLocalDev: createConfig("specularLocalDev"),
+    specularDev: createConfig("specularDev"),
```

Comment:
can you add comments explaining these

Reply:
localdev is the specular network while localhost is the local l1; will add comments